src/stockspanel.cpp

```diff
@@ -444,7 +444,10 @@ wxString StocksListCtrl::getStockInfo(int selectedIndex) const
 
     double stocktotalDifference = stockCurrentPrice - stockavgPurchasePrice;
     //Commision don't calculates here
-    double stockPercentage = (stockCurrentPrice / stockPurchasePrice - 1.0)*100.0;
+    const wxString& stockPercentage = (stockPurchasePrice != 0.0)
+        ? wxString::Format("(%s %%)", Model_Currency::toStringNoFormatting(
```

Comment:
@vomikan MacOS clang complains: 
`/Users/mark/gitrepos/moneymanagerex/src/stockspanel.cpp:448:9: error: conditional expression is ambiguous; 'wxString' can be converted to 'const wxChar *' (aka 'const wchar_t *') and vice versa
        ? wxString::Format("(%s %%)", Model_Currency::toStringNoFormatting(`